config/settings.py

```diff
@@ -166,6 +167,15 @@
 ADMIN_URL = os.getenv("ADMIN_URL", "admin/")
 
 # LOGIN and OIDC provider options
+AUTHENTICATION_BACKENDS = ("mozilla_django_oidc.auth.OIDCAuthenticationBackend",)
```

Comment:
Pour que le login sur l'admin fonctionne toujours, on devra ajouter ici le backend par défaut. Cf. [question sur stackoverflow](https://stackoverflow.com/questions/59881651/django-mozilla-django-oidc-and-admin)

```python
AUTHENTICATION_BACKENDS = [
    'django.contrib.auth.backends.ModelBackend',
    'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
]
```